src/modules/m_filter.cpp

```diff
@@ -882,19 +882,21 @@ void ModuleFilter::ReadFilters()
 	ConfigTagList tags = ServerInstance->Config->ConfTags("keyword");
 	for (ConfigIter i = tags.first; i != tags.second; ++i)
 	{
-		std::string pattern = i->second->getString("pattern");
-		std::string reason = i->second->getString("reason");
-		std::string action = i->second->getString("action");
-		std::string flgs = i->second->getString("flags");
-		unsigned long duration = i->second->getDuration("duration", 10*60, 1);
-		if (flgs.empty())
-			flgs = "*";
+		ConfigTag* tag = i->second;
+		std::string pattern = tag->getString("pattern");
+		std::string reason = tag->getString("reason");
+		std::string action = tag->getString("action");
+		std::string flags = tag->getString("flags");
```

Comment:
Here you could do:

```suggestion
		std::string flags = tag->getString("flags", "*", 1);
```

... and then get rid of the flags.empty() check.

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4